clients/cli/src/main/java/org/apache/gravitino/cli/outputs/PlainFormat.java

```diff
@@ -33,6 +33,8 @@ public static void output(Object object) {
       new MetalakesStringFormat().output((Metalake[]) object);
     } else if (object instanceof Catalog) {
       new CatalogStringFormat().output((Catalog) object);
+    } else if (object instanceof Catalog[]) {
+      new CatalogsStringFormat().output((Catalog[]) object);
```

Comment:
Please change `CatalogsStringFormat` to `CatalogsPlainFormat`,  and `MetalakesStringFormat `, `CatalogStringFormat ` and other `*StringFormat` class

Reply:
I wodl call them `CatalogStringFormat`, `CatalogPlainFormat` etc

Reply:
The `CatalogPlainFormat` output `Catalog` object.
The `CatalogsPlainFormat` output `Catalog[]` objects.
...